src/main/java/nl/mrwouter/minetopiafarms/commands/MTFarmsCMD.java

```diff
@@ -18,7 +21,7 @@
 import nl.mrwouter.minetopiafarms.utils.Updat3r;
 import nl.mrwouter.minetopiafarms.utils.Utils;
 
-public class MTFarmsCMD implements CommandExecutor {
+public class MTFarmsCMD implements CommandExecutor, TabCompleter {
```

Comment:
Niet nodig, maar het updateinfo argument mag er sowieso uit, Is iets wat voor testing gebruikt is maar verder eigenlijk niet relevant